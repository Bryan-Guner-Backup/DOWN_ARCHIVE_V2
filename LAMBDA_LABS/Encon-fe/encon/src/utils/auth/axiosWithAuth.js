import axios from 'axios';

export const axiosWithAuth = () => {
  const AUTH_TOKEN = localStorage.getItem('AUTH_TOKEN');
  return axios.create({
    baseURL: 'https://encon-be.herokuapp.com/api',
    headers: {
      'Content-Type': 'application/json',
      Authorization: AUTH_TOKEN,
    },
  });
};
