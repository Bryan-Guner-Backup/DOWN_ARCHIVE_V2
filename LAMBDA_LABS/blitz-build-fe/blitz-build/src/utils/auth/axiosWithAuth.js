import axios from "axios";

//http://44.233.184.65:3334
export const axiosWithAuth = () => {

    return axios.create({
        baseURL:"https://backend.blitzbuild.construction",
        headers: {
            'Access-Control-Allow-Origin': "*",
            user_id: localStorage.getItem("user_id"),
            id_token: localStorage.getItem("id_token"),
            project_name:localStorage.getItem("project_name")
        }
    })
};