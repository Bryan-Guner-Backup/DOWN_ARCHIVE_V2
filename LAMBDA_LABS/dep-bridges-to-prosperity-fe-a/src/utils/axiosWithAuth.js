import axios from 'axios';

// DEFINING AXIOS WITH AUTH FUNC
export const axiosWithAuth = idToken => {
  const token = idToken;

  return axios.create({
    headers: {
      Authorization: `Bearer ${token}`,
    },
    baseURL: `${process.env.REACT_APP_API_URI}`,
  });
};
