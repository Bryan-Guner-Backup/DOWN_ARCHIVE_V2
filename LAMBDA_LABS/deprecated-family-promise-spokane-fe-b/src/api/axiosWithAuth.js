import axios from 'axios';
const environment = process.env.ENV || 'development';

const ApiUrl =
  environment !== 'development'
    ? process.env.REACT_APP_API_URI
    : 'http://localhost:8000/';

// needs to be set in Amplify you want this to work
// set to your "Product Node Server URI"... heroku app
export const axiosWithAuth = () => {
  const token = JSON.parse(localStorage.getItem('okta-token-storage'))?.idToken
    ?.value;
  return axios.create({
    headers: {
      Authorization: `Bearer ${token}`,
    },

    baseURL: ApiUrl,

    // baseURL: 'https://family-promise-spokane-be-b.herokuapp.com/',
    // baseURL: 'https://family-pomise-spokane.herokuapp.com',

  });
};
