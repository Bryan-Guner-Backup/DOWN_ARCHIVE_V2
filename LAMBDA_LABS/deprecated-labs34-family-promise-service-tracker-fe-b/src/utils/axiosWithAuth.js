import axios from 'axios';

export const axiosWithAuth = () => {
  const tokenObj = JSON.parse(localStorage.getItem('okta-token-storage'));
  let token = tokenObj.idToken.idToken;
  return axios.create({
    headers: {
      Authorization: `Bearer ${token}`,
    },
    baseURL: 'https://fp-service-tracker-b.herokuapp.com',
  });
};
