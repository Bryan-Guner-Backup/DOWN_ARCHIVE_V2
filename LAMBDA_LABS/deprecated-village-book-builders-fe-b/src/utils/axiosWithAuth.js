import axios from 'axios';

export const axiosWithAuth = () => {
  // console.log('axios with auth ran here');
  const token = window.localStorage.getItem('token');

  // console.log("AxiosAuth --> baseURL:", process.env.REACT_APP_BASEURL);
  // console.log("AxiosAuth --> token:", token);

  return axios.create({
    baseURL: process.env.REACT_APP_BASE_URL,
    headers: {
      Authorization: token,
    },
  });
};
