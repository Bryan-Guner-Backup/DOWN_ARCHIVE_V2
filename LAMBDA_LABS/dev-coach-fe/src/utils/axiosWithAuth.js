import axios from 'axios';

export default function axiosWithAuth() {
  const token =
    localStorage.getItem('token') || localStorage.getItem('tempuser');
  const axiosInstance = axios.create({
    baseURL: process.env.LOCAL_BACKEND_SERVER,
    headers: {
      'Content-Type': 'application/json',
      authorization: token,
    },
  });
  return axiosInstance;
}
