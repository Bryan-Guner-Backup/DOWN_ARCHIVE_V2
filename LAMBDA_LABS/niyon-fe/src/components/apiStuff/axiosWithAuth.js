import axios from 'axios'
import io from 'socket.io-client'

const token = localStorage.getItem('token')
export const axiosWithAuth = () => {
  return axios.create({
    baseURL: 'https://niyon-app.herokuapp.com',
    headers: {
      authorization: token
    }
  })
}

const createSocket = () => {
  return axios.create({
    baseURL: 'https://niyon-be-chat.herokuapp.com'
    // headers: {
    //   authorization: token,
    // },
  })
}

export const socketIO = () => {
  return io.connect('https://niyon-be-chat.herokuapp.com')
}
export const getChatHistory = async (roomName) => {
  console.log('axios call>>>>', roomName)
  return await createSocket().get(`/chathistory?room_name=${roomName}`)
}

export const getProfile = (id) => {
  return axiosWithAuth().get(`/profile/${id}`)
}

export const saveProfile = (id, profile) => {
  return axiosWithAuth().post(`/profile/${id}`, profile)
}

export const searchProfile = () => {
  return axiosWithAuth().get('/profile')
}

export const logIn = (values) => {
  return axiosWithAuth().post('/auth/login', values)
}

export const registerUser = (userDetails) => {
  return axiosWithAuth().post('/auth/register', userDetails)
}

export const getNewsFeed = () => {
  return axiosWithAuth().get('/news')
}

export const getUserCard = (endPoint, id, payload) => {
  return axiosWithAuth().post(`/connection/${endPoint}/${id}`, payload)
}
