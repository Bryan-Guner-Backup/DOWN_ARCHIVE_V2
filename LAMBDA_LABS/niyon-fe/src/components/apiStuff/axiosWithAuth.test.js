import toJSON from 'enzyme-to-json'
import { axiosWithAuth } from './axiosWithAuth'

describe('checking axiosWithAuth', () => {
  const mockedObj = { axiosWithAuth }
  let mockAxios
  beforeEach(() => {
    mockAxios = jest.spyOn(mockedObj, 'axiosWithAuth')
  })

  afterEach(() => {
    jest.clearAllMocks()
  })
  it('should pass snapshot testing', () => {
    expect(toJSON(axiosWithAuth)).toMatchSnapshot()
  })

  it('axiosWithAuth should be used', () => {
    mockedObj.axiosWithAuth()
    expect(mockAxios).toHaveBeenCalled()
  })
})
