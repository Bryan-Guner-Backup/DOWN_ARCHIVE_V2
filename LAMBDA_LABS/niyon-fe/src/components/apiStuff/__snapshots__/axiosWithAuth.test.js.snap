// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checking axiosWithAuth should pass snapshot testing 1`] = `undefined`;
